packages/theme-patternfly-org/components/tableOfContents/tableOfContents.js

```diff
@@ -10,25 +10,57 @@ export const TableOfContents = ({ items }) => {
     const { innerWidth } = window;
     innerWidth !== width && setWidth(innerWidth);
   }
+  let wasSublistRendered = false;
 
-  const renderItem = (item, index) => {
-    return Array.isArray(item)
-      ? (
-        <JumpLinksList key={index} className="ws-toc-sublist">
-          {item.map(renderItem)}
+  const renderSublist = (item, nextItem, idx) => {
```

Comment:
For this, does `idx` need to be utilized now that the key attribute is no longer using the index, or could the index be omitted from the params here?

Reply:
You're correct, this was leftover from previous refactoring - removed this unused param 👍